### PR TITLE
Feature redirect

### PR DESCRIPTION
**Pull request by mrfishball**

### CODE REVIEW

**Comment by ericmeyer** [comment]
Good set of commits.

File: Sources/SwerverCore/Response/ResponseHeader.swift [hunk 1]
```diff
@@ -3,4 +3,5 @@ public enum ResponseHeader: String {
     case contentLength = "Content-Length: "
     case allow = "Allow: "
     case date = "Date: "
+    case location = "Location: "
```
Comment (ericmeyer) [inline Sources/SwerverCore/Response/ResponseHeader.swift]: I would refactor these to not include `": "`.  Joining the header key/value pairs with a `": "` is part of formatting the response.

File: Tests/SwerverTests/ActionSpec/RedirectActionSpec.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class RedirectActionSpec: QuickSpec {
+    
+    override func spec() {
```
Comment (ericmeyer) [inline Tests/SwerverTests/ActionSpec/RedirectActionSpec.swift:8]: Make sure to revisit theses tests and incorporate the things we talked about during pairing today.

**Review by ericmeyer — CHANGES_REQUESTED** [review]
In general it looks good. The main things that I would comment on are things either:
1. Covered in other PRs
2. Covered during today's pairing session

File: Sources/SwerverCore/Response/ResponseHeader.swift (hunk 1 above)
Comment (mrfishball) [reply Sources/SwerverCore/Response/ResponseHeader.swift]: will remove the ": "

File: Tests/SwerverTests/ActionSpec/RedirectActionSpec.swift (hunk 2 above)
Comment (mrfishball) [reply Tests/SwerverTests/ActionSpec/RedirectActionSpec.swift:8]: will check that

File: Sources/SwerverCore/Response/ResponseHeader.swift (hunk 1 above)
Comment (mrfishball) [reply Sources/SwerverCore/Response/ResponseHeader.swift]: fixed

**Review by cmvandrevala — APPROVED** [review]

File: Tests/SwerverTests/ActionSpec/RedirectActionSpec.swift (hunk 2 above)
Comment (mrfishball) [reply Tests/SwerverTests/ActionSpec/RedirectActionSpec.swift:8]: improvement implemented as discussed on Firday

File: Tests/SwerverTests/RouterSpec/RouterSpec.swift [hunk 3]
```diff
@@ -11,6 +11,7 @@ class RouterSpec: QuickSpec {
             guard let urlOne = URL(string: Resource.test_get.rawValue),
```
Comment (ericmeyer) [inline Tests/SwerverTests/RouterSpec/RouterSpec.swift]: These tests are still testing via the old style of tests. Don't forget to update these to match how we refactored tests on Friday.

File: Tests/SwerverTests/TestData.swift [hunk 4]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (ericmeyer) [inline Tests/SwerverTests/TestData.swift]: This needs to be removed. This is a good reminder to make sure to run your tests before committing.

**Review by ericmeyer — CHANGES_REQUESTED** [review]

File: Tests/SwerverTests/RouterSpec/RouterSpec.swift (hunk 3 above)
Comment (mrfishball) [reply Tests/SwerverTests/RouterSpec/RouterSpec.swift]: will take a look

File: Tests/SwerverTests/TestData.swift (hunk 4 above)
Comment (mrfishball) [reply Tests/SwerverTests/TestData.swift]: will fix that

File: Tests/SwerverTests/RouterSpec/RouterSpec.swift (hunk 3 above)
Comment (mrfishball) [reply Tests/SwerverTests/RouterSpec/RouterSpec.swift]: fixed

File: Tests/SwerverTests/TestData.swift (hunk 4 above)
Comment (mrfishball) [reply Tests/SwerverTests/TestData.swift]: fixed

**Review by ericmeyer — APPROVED** [review]

**Pull request merged**